.github/workflows/aws-snp-launchmeasurement.yml

```diff
@@ -52,7 +52,7 @@ jobs:
 
     - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
       with:
-        repository: edgelesssys/sev-snp-measure-go.git
+        repository: virtee/sev-snp-measure-go.git
         ref: main
```

Comment:
should we pin this to a commit hash given that we aren't the only ones anymore that can commit to the repo?